packages/fluent-theme/src/styles/injectStyle.ts

```diff
@@ -0,0 +1,9 @@
+export const injectedStyles = '@--INJECTED-STYLES-CONTENT--@';
+
+export default function injectStyles() {
+  if (globalThis.document) {
+    const style = document.createElement('style');
```

Comment:
We will need to add `nonce` attribute.

I expect a prop from the upstreamer.

```ts
root.render(
  <FluentThemeProvider nonce="a1b2c3d">
     ...
  </FluentThemeProvider>
);
```

And then we grab the nonce and inject it here.

You can do it in another PR.

Reply:
Sure, what should it do?